test/cluster/lwt/test_lwt_during_teblets_resize.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+
+import asyncio
+import logging
+import os
+import random
+from typing import Optional
+
+import pytest
+from cassandra import ConsistencyLevel
+from cassandra import WriteTimeout, OperationTimedOut
+from cassandra.query import PreparedStatement, SimpleStatement
+
+from test.cluster.conftest import skip_mode
+from test.cluster.util import new_test_keyspace
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error_one_shot
+
+from test.cluster.lwt.lwt_common import (
+    BaseLWTTester,
+    wait_for_tablet_count,
+    DEFAULT_WORKERS,
+    DEFAULT_NUM_KEYS,
+)
+
+from test.pylib.tablets import get_tablet_count
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+@pytest.mark.asyncio
+@skip_mode("release", "error injections are not supported in release mode")
+@skip_mode("debug", "debug mode is too slow for this test")
+async def test_multi_column_lwt_during_split_merge(manager: ManagerClient):
+    """Test multi-column LWT pattern during tablet split and merge operations"""
+
+    cfg = {
+        "enable_tablets": True,
+        "tablet_load_stats_refresh_interval_in_seconds": 1,
+        "enable_tablet_merging": True,
+        "target-tablet-size-in-bytes": 1024 * 16,
+        "rf_rack_valid_keyspaces": False,
+    }
+
+    servers = await manager.servers_add(6, config=cfg)
+    # Disable balancer initially
+    for s in servers:
+        await manager.api.disable_tablet_balancing(s.ip_addr)
```

Comment:
disable_tablet_balancing() affects global state, sufficient to call on one node.